x-pack/plugins/elastic_assistant/server/routes/knowledge_base/entries/bulk_actions_route.ts

```diff
@@ -289,14 +301,18 @@ export const bulkActionKnowledgeBaseEntriesRoute = (router: ElasticAssistantPlug
                 })
               : undefined;
 
-          return buildBulkResponse(response, {
-            // @ts-ignore-next-line TS2322
-            updated: transformESToKnowledgeBase(docsUpdated),
-            created: created?.data ? transformESSearchToKnowledgeBaseEntry(created?.data) : [],
-            deleted: docsDeleted ?? [],
-            skipped: [],
-            errors,
-          });
+          return buildBulkResponse(
+            response,
+            {
+              // @ts-ignore-next-line TS2322
+              updated: transformESToKnowledgeBase(docsUpdated),
```

Comment:
Maybe we could also add telemetry for `update` and `delete` operations.

Reply:
Also, would be nice to report errors during the `bulk` call.

Reply:
Do we have a separate task for these operations?

Reply:
It seems we're not interested in `update` or `delete`, I asked Pedro and was told that wasn't important to track